src/Metadata/ApiResource.php

```diff
@@ -330,8 +330,12 @@ public function withDescription(?string $description = null): self
         return $self;
     }
 
-    public function getTypes(): array
+    public function getTypes(): ?array
     {
+        if (!$this->types) {
+            $this->types = [$this->shortName];
+        }
+
```

Comment:
revert this there should not be any logic here

Reply:
Oups sorry it wasn't supposed to be pushed ^^'